src/Icons/Controllers/ChangePasswordUriController.cs

```diff
@@ -0,0 +1,94 @@
+﻿using Bit.Icons.Models;
+using Bit.Icons.Services;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Caching.Memory;
+
+namespace Bit.Icons.Controllers;
+
+[Route("change-password-uri")]
+public class ChangePasswordUriController : Controller
+{
+    private readonly IMemoryCache _memoryCache;
+    private readonly IDomainMappingService _domainMappingService;
+    private readonly IChangePasswordUriService _changePasswordService;
+    private readonly ChangePasswordUriSettings _changePasswordSettings;
+    private readonly ILogger<ChangePasswordUriController> _logger;
+
+    public ChangePasswordUriController(
+        IMemoryCache memoryCache,
+        IDomainMappingService domainMappingService,
+        IChangePasswordUriService changePasswordService,
+        ChangePasswordUriSettings iconsSettings,
+        ILogger<ChangePasswordUriController> logger)
+    {
+        _memoryCache = memoryCache;
+        _domainMappingService = domainMappingService;
+        _changePasswordService = changePasswordService;
+        _changePasswordSettings = iconsSettings;
```

Comment:
⛏️ Left over "icon" name 
```suggestion
        ChangePasswordUriSettings changePasswordSettings,
        ILogger<ChangePasswordUriController> logger)
    {
        _memoryCache = memoryCache;
        _domainMappingService = domainMappingService;
        _changePasswordService = changePasswordService;
        _changePasswordSettings = changePasswordSettings;
```

Reply:
copy pasta 🤦 [bee55d7](https://github.com/bitwarden/server/pull/5845/commits/bee55d7fd2686f17a0eab7b1516d99cb20561ff8)